pkg/operator2/operator.go

```diff
@@ -280,3 +291,14 @@ func getPrefixFilter() controller.Filter {
 		DeleteFunc: prefix,
 	}
 }
+
+func (c *authOperator) getOperatorDeploymentPodTemplateHash() (string, error) {
```

Comment:
I think this is saying, if the deployment of the operator changes at all, redeploy the OAuth server.  Can we not just use the RV of the deployment?  And I believe we could have that fed into us via an env var.

Reply:
> And I believe we could have that fed into us via an env var.

hmm I do not think that would work because it comes from the pod and not the deployment

Reply:
Yep, I learn about `EnvVarSource` this way but we probably still won't be able to use it :slightly_frowning_face: